src/json.human.js

```diff
@@ -126,9 +128,28 @@
 
         switch (type) {
         case BOOL:
-            result = data ? sn("span", BOOL_CLASS_NAME, "true")
-                          : sn("span", BOOL_CLASS_NAME, "false");
+            var boolOpt = options.bool;
+            container = document.createElement('div');
+
+            if(boolOpt.showImage) {
+                var img = document.createElement('img');
+                img.setAttribute('class', BOOL_IMAGEE);
+
+                img.setAttribute('src',
+                  '' + (data ? boolOpt.img.true : boolOpt.img.false));
+
+                container.appendChild(img);
+            }
+
+            if(boolOpt.showText){
```

Comment:
appending the image and the text if both are true is an intended feature? otherwise better to use else if (I'm not against the feature, just asking :D)